src/libcrun/cgroup.c

```diff
@@ -802,6 +904,33 @@ int enter_systemd_cgroup_scope (runtime_spec_schema_config_linux_resources *reso
         }
     }
 
+  if (annotations)
+    {
+      size_t prefix_len = sizeof (SYSTEMD_PROPERTY_PREFIX) - 1;
+      size_t i;
+
+      for (i = 0; i < annotations->len; i++)
+        {
+          if (has_prefix (annotations->keys[i], "org.systemd.property."))
```

Comment:
`s/"org.systemd.property."/SYSTEMD_PROPERTY_PREFIX/`

Reply:
This is just a minor styling issue, but I'd rather have
```c
if (!has_prefix(annotations->keys[i], SYSTEMD_PROPERTY_PREFIX))
    continue
```

to improve readability and reduce the levels of indentation